runtime/hash.ll

```diff
@@ -56,12 +56,12 @@ constant:
 block:
   %arghdrptr = getelementptr inbounds %block, %block* %arg, i64 0, i32 0, i32 0
   %arghdr = load i64, i64* %arghdrptr
-  call void @add_hash64(i8* %hasher, i64 %arghdr)
   %arglayout = lshr i64 %arghdr, 48
   %isString = icmp eq i64 %arglayout, 0
   br i1 %isString, label %hashString, label %hashChildren
 hashString:
-  %arglen = and i64 %arghdr, 281474976710655
+  %arglen = and i64 %arghdr, 70368744177663
```

Comment:
Can this value be documented? (I know it's harder to abstract out a constant in a `.ll` file)